pkg/cmd/spks.go

```diff
@@ -0,0 +1,209 @@
+				return err
+			}
+			mariadbPremium, err := QueryPrometheus(ctx, v1api, prometheusQueryArr[1], logger)
+			if err != nil {
+				logger.Error(err, "Error querying Prometheus")
+				return err
+			}
+			redisStandard, err := QueryPrometheus(ctx, v1api, prometheusQueryArr[2], logger)
+			if err != nil {
+				logger.Error(err, "Error querying Prometheus")
+				return err
+			}
+			redisPremium, err := QueryPrometheus(ctx, v1api, prometheusQueryArr[3], logger)
+			if err != nil {
+				logger.Error(err, "Error querying Prometheus")
+				return err
+			}
+
+			odooClient := odoo.NewOdooAPIClient(c.Context, odooURL, odooOauthTokenURL, odooClientId, odooClientSecret, logger)
+			location, err := time.LoadLocation("Europe/Zurich")
+			if err != nil {
+				return fmt.Errorf("load loaction: %w", err)
+			}
+
+			from := time.Now().In(location).UTC()
+			to := time.Now().In(location).Add(-(time.Hour * 24)).UTC()
+
+			billingRecords := []odoo.OdooMeteredBillingRecord{
+				{
+					ProductID:            "appcat-spks-mariadb-standard",
+					InstanceID:           "mariadb-standard",
+					ItemDescription:      "appcat-spks-mariadb-standard",
+					ItemGroupDescription: "SPKS",
+					SalesOrder:           salesOrder,
+					UnitID:               UnitID,
+					ConsumedUnits:        float64(mariadbStandard),
+					TimeRange: odoo.TimeRange{
+						From: from,
+						To:   to,
+					},
+				},
+				{
+					ProductID:            "appcat-spks-mariadb-premium",
+					InstanceID:           "mariadb-premium",
+					ItemDescription:      "appcat-spks-mariadb-premium",
+					ItemGroupDescription: "SPKS",
+					SalesOrder:           salesOrder,
+					UnitID:               UnitID,
+					ConsumedUnits:        float64(mariadbPremium),
+					TimeRange: odoo.TimeRange{
+						From: from,
+						To:   to,
+					},
+				},
+				{
+					ProductID:            "appcat-spks-redis-standard",
+					InstanceID:           "redis-standard",
+					ItemDescription:      "appcat-spks-redis-standard",
+					ItemGroupDescription: "SPKS",
+					SalesOrder:           salesOrder,
+					UnitID:               UnitID,
+					ConsumedUnits:        float64(redisStandard),
+					TimeRange: odoo.TimeRange{
+						From: from,
+						To:   to,
+					},
+				},
+				{
+					ProductID:            "appcat-spks-redis-premium",
+					InstanceID:           "redis-premium",
+					ItemDescription:      "appcat-spks-redis-premium",
+					ItemGroupDescription: "SPKS",
+					SalesOrder:           salesOrder,
+					UnitID:               UnitID,
+					ConsumedUnits:        float64(redisPremium),
+					TimeRange: odoo.TimeRange{
+						From: from,
+						To:   to,
+					},
+				},
+			}
+
+			ticker := time.NewTicker(24 * time.Hour)
+			sigint := make(chan os.Signal, 1)
+			signal.Notify(sigint, os.Interrupt)
+
+			for loop := true; loop; {
+				select {
+				case <-ticker.C:
+					err = odooClient.SendData(billingRecords)
+					if err != nil {
+						logger.Error(err, "can't export data to Odoo")
+						panic(err)
```

Comment:
Please don't use panic here and return the error. This way the error is handled by the cli library.